### PR TITLE
Adding chomp planning pipeline

### PR DESCRIPTION
**Pull request by patelmayank2092**

### CODE REVIEW

File: cob_moveit_bringup/launch/chomp_planning_pipeline.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<launch>
+
+   <!-- CHOMP Plugin for MoveIt! -->
+   <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+   <arg name="planning_plugin" value="chomp_interface/CHOMPPlanner" />
```
Comment (fmessmer) [inline cob_moveit_bringup/launch/chomp_planning_pipeline.xml:5]: using `chomp_interface` would require to define an `<exec_depend>` to `chomp_interface`

@ipa-bfb 
is this package installable from debs?

**Review by fmessmer — CHANGES_REQUESTED** [review]

**Review by fmessmer — APPROVED** [review]
thx

**Comment by mathias-luedtke** [comment]
Perhaps http://wiki.ros.org/chomp_motion_planner

File: cob_manipulation/package.xml [hunk 2]
```diff
@@ -23,6 +23,7 @@
   <exec_depend>cob_moveit_interface</exec_depend>
   <exec_depend>cob_obstacle_distance_moveit</exec_depend>
   <exec_depend>cob_pick_place_action</exec_depend>
+  <exec_depend>moveit_planner_chomp</exec_depend>
```
Comment (mathias-luedtke) [inline cob_manipulation/package.xml:26]: The dependency was added to the meta-package..

**Pull request merged**